tools/gfx/vulkan/render-vk.cpp

```diff
@@ -8740,7 +8748,10 @@ Result VKDevice::getFormatSupportedResourceStates(Format format, ResourceStateSe
     }
     // AccelerationStructure
     if (bufferFeatures & VK_FORMAT_FEATURE_ACCELERATION_STRUCTURE_VERTEX_BUFFER_BIT_KHR)
+    {
         allowedStates.add(ResourceState::AccelerationStructure);
```

Comment:
It seems like the `AccelerationStructure` state (the one that was in this code originally) is actually the one that *doesn't* belong here. Might be good to confirm.